src/nginx/servers.conf.erb

```diff
@@ -0,0 +1,114 @@
+# ERB templated nginx configuration
```

Comment:
This file is a configuration file specific to PAAS deployment. I think it should not be in the `src` directory.